libs/generators/src/lib/generators/generators.component.scss

```diff
@@ -0,0 +1,6 @@
+:host {
+    .container {
```

Comment:
DRY, extract class to the global styles and reuse it

Reply:
@ronnetzer I would happily do that, but we agreed not to use atomic classes (like h-100 or mrg-x-4 and etc...)

Reply:
I'm not so sure if it qualifies as an atomic class, but in any case if its done twice, than it should be extracted and reused